pkg/infra/game/decision/decision.go

```diff
@@ -19,3 +19,18 @@ const (
 	Cower
 	Undecided
 )
+
+type VotingManifesto struct {
+	Running         bool
+	FightImposition bool
+	LootImposition  bool
+	ResignThreshold uint
+	TermLength      uint
+}
+
+type VotingStrategy uint
+
+const (
+	ConfidenceVote uint = iota
```

Comment:
This needs to be taken out of here as a confidence vote isn't an election strategy but will do that tmrw.